packages/design-system-react/src/components/button-secondary/ButtonSecondary.stories.tsx

```diff
@@ -124,7 +124,7 @@ export const IsInverse: Story = {
         Inverse Button
       </ButtonSecondary>
       <ButtonSecondary {...args} isInverse isDanger>
-        No Inverse Danger Button
+        Inverse Danger Button
```

Comment:
Updating Button Secondary to have an isInverse isDanger variant that matches ButtonPrimary